pkg/apis/componentconfig/types.go

```diff
@@ -62,3 +62,213 @@ const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
+
+type KubeletConfiguration struct {
+	// config is the path to the config file or directory of files
+	Config string `json:"config"`
```

Comment:
looks like this matches the flag ordering.  are the flags grouped logically, or why are they ordered this way?